utils/gen-snapshot-tests.js

```diff
@@ -92,7 +91,7 @@ function generateTestFile(root, examples, componentFileMap) {
 
     const outPath = path.join(root, "generated-snapshot.test.js");
     fs.writeFileSync(outPath, data, "utf8");
-    console.log(`wrote ${outPath}`);
+    console.log(`wrote ${outPath}`); // eslint-disable-line no-console
```

Comment:
In the past I've had better luck using `eslint-disable-next-line` as Prettier will keep it intact and won't trap to wrap it. Might be worth trying that?